docs/section22/Rule22-26.md

```diff
@@ -24,7 +24,7 @@
 
 - **Check 1:** Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
 
-  - Check if B-RMR is modeled with at least one air-side system that is Type-11.1, 11.2 or 11b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-11.1", "SYS-11.2", "SYS-11B"]): NEXT_APPLICABILITY_CHECK`
+  - Check if B-RMR is modeled with at least one air-side system that is Type-11.1, 11.2 or 11.1b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-11.1", "SYS-11.2", "SYS-11B"]): NEXT_APPLICABILITY_CHECK`
   
   - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
 
```

Comment:
The RDS Rule Logic says to create a dictionary `loop_pump_dictionary` but it doesn't seem to use it.

Reply:
Right, I delete this section in the RDS.